international-journal-of-nuclear-security.csl

```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>International Journal of Nuclear Security</title>
+    <id>http://www.zotero.org/styles/international-journal-of-nuclear-security</id>
+    <link href="http://www.zotero.org/styles/international-journal-of-nuclear-security" rel="self"/>
+    <link href="http://www.sciencemag.org/about/authors/prep/res/refs.dtl" rel="documentation"/>
```

Comment:
do you have your own author guidelines? If so, link to those.
If you're actually just following *Science*, we'd want to make this a dependent style.